src/plugin/cesium/cesiumrenderer.js

```diff
@@ -108,6 +129,46 @@ plugin.cesium.CesiumRenderer.prototype.initialize = function() {
     return new goog.Promise(function(resolve, reject) {
       plugin.cesium.loadCesium().then(function() {
         try {
+          // register the default set of synchronizers
```

Comment:
Moving all of this code to this function makes me slightly nervous. Is there ever a case where `initialize` gets called again in a single session? Calls like these just fit better in a plugin init call where we are pretty certain they won't happen again.

Reply:
The check at the top of this function is essentially a more robust version of `this.isInitialized()` so I think it will be fine. In a basic test, I wasn't able to get it to call twice.